### PR TITLE
Add fixture dictionaries for software profile

### PR DESCRIPTION
**Pull request by HarshvMahawar**
targets #432

### CODE REVIEW

File: tests/spdx3/fixtures.py [hunk 1]
```diff
@@ -298,6 +309,42 @@ def namespace_map_fixture(
...
+    "originated_by": ["https://spdx.test/tools-python/originatedBy"],
+    "supplied_by": ["https://spdx.test/tools-python/suppliedBy"],
+    "built_time": datetime(2004, 1, 1),
+    "release_time": datetime(2005, 1, 1),
+    "valid_until_time": datetime(2006, 1, 1),
+    "standard": ["https://spdx.test/tools-python/standard"],
+}
+
+SOFTWARE_ARTIFACT_DICT = {
+    "content_identifier": "contentIidentifier",
```
Comment (armintaenzertng) [inline tests/spdx3/fixtures.py:322]: `content_identifier` must be a URI, see [here](https://github.com/spdx/spdx-3-model/blob/main/model/Software/Properties/contentIdentifier.md)
```suggestion
    "content_identifier": ""https://spdx.test/tools-python/contentIdentifier"",
```

File: tests/spdx3/fixtures.py [hunk 2]
```diff
@@ -298,6 +309,42 @@ def namespace_map_fixture(
...
+    "purpose": SoftwarePurpose.OTHER,
+    "concluded_license": LicenseField,
+    "declared_license": LicenseField,
```
Comment (armintaenzertng) [inline tests/spdx3/fixtures.py:325]: This is not a correct value for these fields. But I suppose you ran into the circle definition of the `License` fixture being referenced in the same dictionary as the software artifact fixtures. I'll think about how to fix this, in the meantime we can merge it like this, I think.

**Review by armintaenzertng — CHANGES_REQUESTED** [review]
Thanks, this looks very good already, I have only one minor remark.

File: tests/spdx3/fixtures.py (hunk 2 above)
Comment (HarshvMahawar) [reply tests/spdx3/fixtures.py:325]: yes

Comment (HarshvMahawar) [reply tests/spdx3/fixtures.py:325]: Also, do the values in SNIPPET_DICT assumed correctly, or need modifications?

Comment (armintaenzertng) [reply tests/spdx3/fixtures.py:325]: They are fine as far as I'm concerned! :)

**Review by armintaenzertng — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**